shell/browser/browser_linux.cc

```diff
@@ -161,7 +161,11 @@ void Browser::ShowAboutPanel() {
     gtk_about_dialog_set_website(dialog, website.c_str());
   if (about_panel_options_.GetString("iconPath", &icon_path)) {
     GError* error = nullptr;
-    GdkPixbuf* icon = gdk_pixbuf_new_from_file(icon_path.c_str(), &error);
+    int width = 64;   // width of about panel icon in pixels
+    int height = 64;  // height of about panel icon in pixels
+    // set preserve_aspect_ratio to true
+    GdkPixbuf* icon = gdk_pixbuf_new_from_file_at_scale(
+        icon_path.c_str(), width, height, true, &error);
```

Comment:
`gdk_pixbuf_new_from_file_at_size()` is a clearer way of doing lines 166-168 (and supported back to gdk-pixbuf 2.4)